### PR TITLE
Adding compliance data schema change log to securescoreControlProfile entity

### PR DESCRIPTION
**Pull request by sateeshkarimilla**

### CODE REVIEW

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/certificationControl.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/certificationControl.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/complianceInformation.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/complianceInformation.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/secureScoreControlStateUpdate.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/secureScoreControlStateUpdate.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C29%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811290140418037-3515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=xBuSZURQif5XCUE%2FkCQWIKPQh6NtlLGLspCOOS1lSdw%3D&st=2018-11-29T01%3A39%3A02Z&se=2018-12-30T01%3A44%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/resources/securescorecontrolprofiles.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Represents a tenant's secure score per control data. By default, it returns all
 |	azureTenantId	|	String	|	GUID string for tenant ID.	|
 |	controlName	|	String	|	Name of the control. |
 |	title	|	String	|	Title of the control.	|
+| complianceInformation | [complianceInformation](#complianceInformation.md) collection | The collection of compliance information associated with secure score control |
```
Comment (jthake-msft) [inline api-reference/beta/resources/securescorecontrolprofiles.md]: This is an invalid link [complianceInformation](#complianceInformation.md) . Pound sign or ".md" should not be there I believe.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/certificationControl.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/certificationControl.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/complianceInformation.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/complianceInformation.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/secureScoreControlStateUpdate.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/secureScoreControlStateUpdate.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C29%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811291729073236-3515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=eQjlHxsFD3Q7FHSikTJ397mcRUEdRDRpVNc8%2BV4HAP4%3D&st=2018-11-29T17%3A27%3A25Z&se=2018-12-30T17%3A32%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/certificationControl.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/certificationControl.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/complianceInformation.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/complianceInformation.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/secureScoreControlStateUpdate.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/secureScoreControlStateUpdate.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C29%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811291730430817-3515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=BknlP9q%2B6%2FTdfVgxJ1cbFItuCAhplDDyIjn9KOVAdSE%3D&st=2018-11-29T17%3A30%3A42Z&se=2018-12-30T17%3A35%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/certificationControl.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/certificationControl.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/complianceInformation.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/complianceInformation.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/secureScoreControlStateUpdate.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/secureScoreControlStateUpdate.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C29%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811291733328276-3515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=v2z1UInbm%2FhnTgHEapgk3i05Zg3J8UUx6U0p%2BXhppXU%3D&st=2018-11-29T17%3A33%3A52Z&se=2018-12-30T17%3A38%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/changelog.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/concepts/changelog.md) | :warning:Warning | | [Details](#user-content-concepts%2Fchangelog.md)
[api-reference/beta/resources/certificationControl.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/certificationControl.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/complianceInformation.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/complianceInformation.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/secureScoreControlStateUpdate.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/secureScoreControlStateUpdate.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md) | :white_check_mark:Succeeded | |

<a name="concepts%2Fchangelog.md"></a>
### [concepts/changelog.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/concepts/changelog.md)
 - **Line 26**: **[Warning]** Invalid file link:(~/../api-reference/beta/resources/dataPolicyOperation).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812042147228141-3515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=q6wdG1ZWEVH9ZSE7hSdkehoRNKeUeMPl84MGtx4UCG8%3D&st=2018-12-04T21%3A45%3A41Z&se=2019-01-04T21%3A50%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/changelog.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/concepts/changelog.md) | :warning:Warning | | [Details](#user-content-concepts%2Fchangelog.md)
[api-reference/beta/resources/certificationControl.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/certificationControl.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/complianceInformation.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/complianceInformation.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/secureScoreControlStateUpdate.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/secureScoreControlStateUpdate.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md) | :white_check_mark:Succeeded | |

<a name="concepts%2Fchangelog.md"></a>
### [concepts/changelog.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/concepts/changelog.md)
 - **Line 26**: **[Warning]** Invalid file link:(~/../api-reference/beta/resources/dataPolicyOperation).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812042245390122-3515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=SqH7dDg6NjmBgQT6HiqbRnyLhh2Bq591uBgSncZ0tk4%3D&st=2018-12-04T22%3A44%3A02Z&se=2019-01-04T22%3A49%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/changelog.md [hunk 2]
```diff
@@ -8,6 +8,14 @@ description: "This changelog covers what's changed in Microsoft Graph, including
...
 
 For details about known issues with Microsoft Graph APIs, see [Known issues](known-issues.md).
+## December 2018
+
+### Security APIs
+
+| **Change type** | **Version** | **Description**              |
+| :-------------- | :---------- | :--------------------------------------- |
+| Addition        | Beta        | Introduced new complex type [complianceInformation](/graph/api/complianceInformation/team?view=graph-rest-beta).|
+| Addition        | Beta        | Introduced new complex type [certificationControl](/graph/api/certificationControl/team?view=graph-rest-beta).|
```
Comment (Lauragra) [inline concepts/changelog.md]: We need to add secureScoreControlStateUpdate to the Changelog table.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[57e1fd4](https://github.com/sateeshkarimilla/microsoft-graph-docs/pull/3515/commits/57e1fd47d818df5e7098b5ce7012d89521f22172)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 20 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[57e1fd4](https://github.com/sateeshkarimilla/microsoft-graph-docs/pull/3515/commits/57e1fd47d818df5e7098b5ce7012d89521f22172)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[57e1fd4](https://github.com/sateeshkarimilla/microsoft-graph-docs/pull/3515/commits/57e1fd47d818df5e7098b5ce7012d89521f22172)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fresources%2Fsecurescorecontrolprofiles.md)
[api-reference/beta/resources/certificationControl.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/certificationControl.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/complianceInformation.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/complianceInformation.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/securescorecontrolstateopdate.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolstateopdate.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/concepts/changelog.md) | :white_check_mark:Succeeded | |

<a name="api-reference%2Fbeta%2Fresources%2Fsecurescorecontrolprofiles.md"></a>
### [api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md)
 - **Line 41**: **[Warning]** Invalid file link:(~/beta/resources/secureScoreControlStateUpdate.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C19%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812191841593640-3515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sIl5ewWNAsCB%2BPTgRowtbS5U4sgFMVln7cvqHgaT4CA%3D&st=2018-12-19T18%3A40%3A47Z&se=2019-01-19T18%3A45%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[005016a](https://github.com/sateeshkarimilla/microsoft-graph-docs/pull/3515/commits/005016a5df564c4c3e73e3d893fcf219f2f15108)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 20 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[005016a](https://github.com/sateeshkarimilla/microsoft-graph-docs/pull/3515/commits/005016a5df564c4c3e73e3d893fcf219f2f15108)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[005016a](https://github.com/sateeshkarimilla/microsoft-graph-docs/pull/3515/commits/005016a5df564c4c3e73e3d893fcf219f2f15108)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fresources%2Fsecurescorecontrolprofiles.md)
[api-reference/beta/resources/certificationControl.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/certificationControl.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/complianceInformation.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/complianceInformation.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/securescorecontrolstateopdate.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolstateopdate.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/concepts/changelog.md) | :white_check_mark:Succeeded | |

<a name="api-reference%2Fbeta%2Fresources%2Fsecurescorecontrolprofiles.md"></a>
### [api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md)
 - **Line 41**: **[Warning]** Invalid file link:(~/beta/resources/secureScoreControlStateUpdate.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C19%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812191842177015-3515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=FjQv1xZYvNidS5hFEGr2ACzJMgBpZiZbjTI%2BW2opF2g%3D&st=2018-12-19T18%3A44%3A07Z&se=2019-01-19T18%3A49%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[0ca093b](https://github.com/sateeshkarimilla/microsoft-graph-docs/pull/3515/commits/0ca093b3eadf06050b6a6886b8a63e9718cc1ebf)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[0ca093b](https://github.com/sateeshkarimilla/microsoft-graph-docs/pull/3515/commits/0ca093b3eadf06050b6a6886b8a63e9718cc1ebf)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fresources%2Fsecurescorecontrolprofiles.md)
[api-reference/beta/resources/certificationControl.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/certificationControl.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/complianceInformation.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/complianceInformation.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/securescorecontrolstateopdate.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolstateopdate.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/concepts/changelog.md) | :white_check_mark:Succeeded | |

<a name="api-reference%2Fbeta%2Fresources%2Fsecurescorecontrolprofiles.md"></a>
### [api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md)
 - **Line 41**: **[Warning]** Invalid file link:(~/beta/resources/securescorecontrolstateupdate.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C19%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812192029489656-3515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=xYzoUo0DTaCBZTt4f2NM%2Fj%2B6bAvcNro1con8TghTgII%3D&st=2018-12-19T20%3A28%3A15Z&se=2019-01-19T20%3A33%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[42c119d](https://github.com/sateeshkarimilla/microsoft-graph-docs/pull/3515/commits/42c119d9327202b5655a4c0d9553134bf8fcae7f)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 20 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[42c119d](https://github.com/sateeshkarimilla/microsoft-graph-docs/pull/3515/commits/42c119d9327202b5655a4c0d9553134bf8fcae7f)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[42c119d](https://github.com/sateeshkarimilla/microsoft-graph-docs/pull/3515/commits/42c119d9327202b5655a4c0d9553134bf8fcae7f)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fresources%2Fsecurescorecontrolprofiles.md)
[api-reference/beta/resources/certificationControl.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/certificationControl.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/complianceInformation.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/complianceInformation.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/securescorecontrolstateopdate.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolstateopdate.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/concepts/changelog.md) | :white_check_mark:Succeeded | |

<a name="api-reference%2Fbeta%2Fresources%2Fsecurescorecontrolprofiles.md"></a>
### [api-reference/beta/resources/securescorecontrolprofiles.md](https://github.com/sateeshkarimilla/microsoft-graph-docs/blob/securescore/api-reference/beta/resources/securescorecontrolprofiles.md)
 - **Line 41**: **[Warning]** Invalid file link:(~/beta/resources/securescorecontrolstateupdate.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C19%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812192031422095-3515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=O7I9XtsNQYwjq6mMS%2Blo9YAxnfA7Vmv4nAY8h6zjcuY%3D&st=2018-12-19T20%3A31%3A42Z&se=2019-01-19T20%3A36%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**